lessons/Assignment-3-Ruby-Part-1.md

```diff
@@ -143,7 +143,7 @@ The program should do the following:
 
 ## Program 4
 
-Create `guesser.rb`.  When this program starts, it asks you to think of a number from 1 to 100.  It then makes guesses.  You tell it whether the guess is too low, too high, or correct, until it guesses your number.  The program will need to keep track of the lowest and highest possible remaining values, and it should guess something between them.  The program should prompt you again if you say something besides "too low", "too high" or "correct".  The program should also handle the case where you are lying, and should call you out.  You are lying if the program correctly guesses the number, but you don't say so.  You are lying if you tell the program that its guess is too low, when it's really too high, or vice versa.  Of course, the program won't be able to tell right away that you are lying, as it will have to exhaust the possible answers.  Make the program efficient -- basically a binary search.  Once it guesses correctly, or figures out you are lying, it should prompt you to play again, exiting if you decline.
+Create `guesser.rb`.  When this program starts, it asks you to think of a number from 1 to 100.  It then makes guesses.  You tell it whether the guess is too low, too high, or correct, until it guesses your number.  The program will need to keep track of the lowest and highest possible remaining values, and it should guess something between them.  The program should prompt you again if you say something besides "too low", "too high" or "correct".  The program should also handle the case where you are lying, and should call you out.  You are lying if the program correctly guesses the number, but you don't say so.  You are lying if you tell the program that its guess is too low, when it's really too high, or vice versa.  Of course, the program won't be able to tell right away that you are lying, as it will have to exhaust the possible answers.  Make the program efficient -- basically a binary search.  Once it guesses correctly, or figures out you are lying, it should ask if you'd like to play again, exiting if you decline.
```

Comment:
GitHub isn't doing a very good job of highlighting the substring that changed, so I'll do it.

Before:
> Once it guesses correctly, or figures out you are lying, it should prompt you to play again, exiting if you decline.

After:
> Once it guesses correctly, or figures out you are lying, it should ask if you'd like to play again, exiting if you decline.

I suspect that the wording around "prompt" was a bit ambiguous because asking for "too low", "too high" or "correct" is technically a prompt, but not a question about playing again or not.